test/src/com/xilinx/rapidwright/rwroute/TestRWRoute.java

```diff
@@ -378,7 +378,7 @@ public void testTimingDrivenPartialRouting() {
         VivadoToolsHelper.assertFullyRouted(design);
     }
 
-    void testSingleConnectionHelper(String partName,
+    Design testSingleConnectionHelper(String partName,
```

Comment:
Should this be made `public static` so it could be useful in other testing contexts?